dev_guide/builds/advanced_build_operations.adoc

```diff
@@ -281,18 +281,17 @@ spec:
   successfulBuildsHistoryLimit: 2 <1>
   failedBuildsHistoryLimit: 2 <2>
 ----
-<1> `successfulBuildsHistoryLimit` will retain up to 2 builds with a status of `completed`
-<2> `failedBuildsHistoryLimit` will retain up to 2 builds with a status of `failed`, `cancelled`, or `error`
+<1> `successfulBuildsHistoryLimit` will retain up to two builds with a status of `completed`.
+<2> `failedBuildsHistoryLimit` will retain up to two builds with a status of `failed`, `cancelled`, or `error`.
 
 Build pruning is triggered by the following actions:
 
- - the build configuration is updated
- - a build completes its lifecycle
+ - Updating a build configuration.
+ - A build completes its lifecycle.
 
 Builds are sorted by their creation timestamp with the oldest builds being pruned first.
 
 [NOTE]
 ====
 Administrators can manually prune builds using the xref:../../admin_guide/pruning_resources.adoc#pruning-builds[oadm object pruning command].
 ====
```

Comment:
@bmcelvee This note box should be ifdef'd for just OCP, Origin, and (maybe?) Dedicated. This would be a broken link for Online since it doesn't have an `admin_guide`.

Also `oadm` could be marked up in the link.